extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -2126,64 +2231,47 @@ describes.realWin('Events', {amp: 1}, (env) => {
     });
 
     describe('should wait on correct readyPromise', () => {
-      const selector = '.target';
-
       it('with waitFor default value', () => {
         // Default case: selector is not specified
-        expect(tracker.getReadyPromise(undefined, undefined)).to.be.null;
+        expect(tracker.getReadyPromise(undefined)).to.be.null;
         // Default case: waitFor is not specified, no AMP element selected
         iniLoadTrackerMock
           .expects('getRootSignal')
           .returns(Promise.resolve())
           .once();
-        const waitForTracker1 = tracker.getReadyPromise(undefined, ':root');
+        const waitForTracker1 = tracker.getReadyPromise('ini-load');
         return waitForTracker1.then(() => {
           iniLoadTrackerMock
             .expects('getElementSignal')
             .withExactArgs('ini-load', target)
             .returns(Promise.resolve())
             .once();
           // Default case: waitFor is not specified, AMP element selected
-          const promise2 = tracker.getReadyPromise(undefined, selector, target);
+          const promise2 = tracker.getReadyPromise('ini-load', target);
```

Comment:
this test seems so unnecessary now, let's remove it.

Reply:
Test was removed as per comment above

Reply:
I still see the test for default case, am i missing anything?

Reply:
Oops, I misunderstood your first comment. Removed the default test :)